src/parser/xasl_generation.c

```diff
@@ -327,7 +328,7 @@ typedef struct corr_info
 FILE *query_Plan_dump_fp = NULL;
 char *query_Plan_dump_filename = NULL;
 
-static XASL_SUPP_INFO xasl_Supp_info = { NULL, NULL, NULL, NULL, 0, 0 };
+static XASL_SUPP_INFO xasl_Supp_info = { NULL, NULL, NULL, NULL, 0, 0, false };
```

Comment:
0, not false

Reply:
Thank you, I missed to change it when I changed the type.
Refected in https://github.com/CUBRID/cubrid/pull/2450/commits/784efac5eabf7bd40748f0362f688d82a0974714